Classes/GraphQL/Resolver/Type/QueryResolver.php

```diff
@@ -174,7 +179,17 @@ protected function createAssetProxyQuery(
             }
         }
 
-        // TODO: Implement sorting via `SupportsSortingInterface`
+        if ($sortBy && $assetProxyRepository instanceof SupportsSortingInterface) {
+            switch ($sortBy) {
```

Comment:
This implementation equals the one from the old media browser. I was thinking of a more flexible solution, e.g. go for `$assetProxyRepository->orderBy([$sortBy => $sortDirection]);` and leave the sortB completely to the frontend, but decided a more explicit version here would be better

Reply:
Yes that's better 👍